MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressPassthru.c

```diff
@@ -516,7 +516,7 @@ NvmExpressPassThru (
   EFI_STATUS                     PreviousStatus;
   EFI_PCI_IO_PROTOCOL            *PciIo;
   NVME_SQ                        *Sq;

```

Comment:
does Sq have the same protentional problem?

Reply:
Per offline discussion with @apop5, added the volatile keyword for SQs as well, more description in edited PR description.